.github/workflows/downgrade.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           version: ${{ matrix.version }}
       - uses: cjdoris/julia-downgrade-compat-action@v1
```

Comment:
@Krastanov it looks like this action was recently archived, see: https://github.com/cjdoris/julia-downgrade-compat-action

Shoud I changed it to:
```julia
julia-actions/julia-downgrade-compat-action@v1
```
I have added all the base libraries to be skipped but it still fails maybe this will help?

Reply:
yes

Reply:
oh, I see you have already done so